Packs/BeyondTrustEPM/ModelingRules/BeyondTrustEPM/BeyondTrustEPM.xif

```diff
@@ -0,0 +1,58 @@
+[MODEL: dataset = beyondtrust_epm_raw]
+alter
+    Get_host_os_type = host -> os.type,
+    Get_EPMWinMac_Event_Action = EPMWinMac -> Event.Action,
+    Get_EPMWinMac_Event_Type = EPMWinMac -> Event.Type,
+    Get_related_ip = related -> ip[],
+    Get_process_command_line = process -> command_line,
+    Get_process_pid = to_integer(process -> pid),
+    Get_process_parent_name = process -> parent.name,
+    Get_file_hash_md5 = file -> hash.md5,
+    Get_file_hash_sha256 = file -> hash.sha256,
+    Get_file_extension = file -> extension,
+    Get_file_path = file -> path,
+    Get_file_name = file -> name,
+    Get_file_directory = file -> directory,
+    Get_file_code_signature_subject_name = file -> code_signature.subject_name,
+    Get_file_code_signature_valid = file -> code_signature.valid,
+    Get_file_code_signature_exists = file -> code_signature.exists
+| alter
+    xdm.source.host.os = Get_host_os_type,
+    xdm.source.host.os_family = if(Get_host_os_type contains "windows", XDM_CONST.OS_FAMILY_WINDOWS, Get_host_os_type contains "mac", XDM_CONST.OS_FAMILY_MACOS),
+    xdm.event.outcome_reason = Get_EPMWinMac_Event_Action,
+    xdm.event.outcome = if(outcome = "success", XDM_CONST.OUTCOME_SUCCESS, action in ("process-start-blocked"), XDM_CONST.OUTCOME_FAILED),
+    xdm.event.type = Get_EPMWinMac_Event_Type,
+    xdm.event.original_event_type = action,
+    xdm.event.id = code,
+    xdm.source.host.hostname = hostName,
+    xdm.source.host.device_id = id,
+    xdm.source.user.username = name,
+    xdm.source.host.ipv4_addresses = arraymap(Get_related_ip, replex("@element", "\"", "")),
+    xdm.source.process.command_line = Get_process_command_line,
+    xdm.source.process.pid = Get_process_pid,
+    xdm.target.file.md5 = Get_file_hash_md5,
+    xdm.target.file.sha256 = Get_file_hash_sha256,
+    xdm.target.file.extension = Get_file_extension,
+    xdm.target.file.path = Get_file_path,
+    xdm.target.file.signer = if(Get_file_code_signature_exists = "true", Get_file_code_signature_subject_name),
```

Comment:
Did we make sure there won't be output FALSE, even when there is no relevant file?